kedro-datasets/kedro_datasets/dask/csv_dataset.py

```diff
@@ -0,0 +1,128 @@
+"""``CSVDataset`` is a data set used to load and save data to CSV files using Dask
+dataframe"""
+
+from copy import deepcopy
+from typing import Any, Dict
+
+import dask.dataframe as dd
+import fsspec
+
+from kedro.io.core import AbstractDataset, get_protocol_and_path
+
+
+# NOTE: kedro.extras.datasets will be removed in Kedro 0.19.0.
+# Any contribution to datasets should be made in kedro-datasets
+# in kedro-plugins (https://github.com/kedro-org/kedro-plugins)
```

Comment:
I'm guessing this is just a copy-paste, but it's no longer relevant so you can remove it.